test/unittests/eof_validation.cpp

```diff
@@ -2,10 +2,38 @@
 // Copyright 2024 The evmone Authors.
 // SPDX-License-Identifier: Apache-2.0
 
+#ifdef _MSC_VER
+// Disable warning C4996: 'getenv': This function or variable may be unsafe.
+#define _CRT_SECURE_NO_WARNINGS
+#endif
+
 #include "eof_validation.hpp"
+#include <test/statetest/statetest.hpp>
+#include <fstream>
+
+namespace fs = std::filesystem;
 
 namespace evmone::test
 {
+namespace
+{
+/// Creates the file path for the exported test based on its name.
+fs::path get_export_test_path(const testing::TestInfo& test_info, std::string_view export_dir)
```

Comment:
I think it make more sense to share this function with `state_transition.cpp`. E.g. `test_export.cpp`?